src/segmantic/seg/transforms.py

```diff
@@ -0,0 +1,82 @@
+from typing import Dict, Optional, Sequence, Union
+
+import torch
+from monai.config.type_definitions import KeysCollection, NdarrayOrTensor
+from monai.transforms.post.array import Ensemble
+from monai.transforms.post.dictionary import Ensembled
+from monai.transforms.transform import Transform
+from monai.utils import TransformBackends
+
+
+class SelectBestEnsemble(Ensemble, Transform):
+    """
+    Execute vote ensemble on the input data.
+    The input data can be a list or tuple of PyTorch Tensor with shape: [C[, H, W, D]],
```

Comment:
this documentation is a copy-paste from VoteEnsemble. Can you adapt it?

Reply:
sure thing will do